asset/quickstart/get-saved-query/getSavedQuery.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+ 
+// [START asset_quickstart_get_saved_query]
+ 
+package GetSavedQueryFunction
+ 
+import (
+	"context"
+	"fmt"
+	"strconv"
+ 
+	asset "cloud.google.com/go/asset/apiv1"
+	"cloud.google.com/go/asset/apiv1/assetpb"
+	cloudresourcemanager "google.golang.org/api/cloudresourcemanager/v1"
+)
+ 
+func getSavedQuery(projectId, savedQueryID string) error {
+	// projectID := "my-project-id"
+	// savedQueryID := "query-ID"
+	ctx := context.Background()
+	client, err := asset.NewClient(ctx)
+	if err != nil {
+		return fmt.Errorf("asset.NewClient: %v", err)
+	}
+	defer client.Close()
+ 
+	cloudResourceManagerClient, err := cloudresourcemanager.NewService(ctx)
```

Comment:
I suspect that you need to have a `defer cloudResourceManagerClient.Close()` after the nil check.

Reply:
as discussed offline, the official guide does not have a close method:) https://pkg.go.dev/google.golang.org/api/cloudresourcemanager/v1